icon-font/css-template.css

```diff
@@ -3,29 +3,33 @@
  * This file is automatically built using a build process
  * If you need to fix errors, see https://github.com/WordPress/dashicons
  */
+
+/* stylelint-disable function-url-quotes, declaration-colon-newline-after */
 <% if (fontfaceStyles) { %>
 <% if (fontSrc1 && embed.length) { %>@font-face {
-	font-family: "<%= fontFamilyName %>";
+	font-family: <%= fontFamilyName %>;
```

Comment:
Similarly via https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#values
> _Quote font names when required_

Font names only require quotes when they contain a space, e.g `"Helvetica Neue"`, not `Helvetica Neue`